python/mxnet/contrib/onnx/onnx2mx/_translation_utils.py

```diff
@@ -245,3 +245,17 @@ def get_input_shape(sym, proto_obj):
     result = mod.get_outputs()[0].asnumpy()
 
     return result.shape
+
+def broadcast_arithmetic_helper(attrs, inputs, proto_obj, current_op_name):
+    """Helper function for broadcast arithmetic ops."""
+    new_attr = {}
+    op_names = ['batchnorm, convolution, deconvolution']
+    if 'broadcast' in attrs and attrs['broadcast'] == 1:
+        broadcast_axis = attrs['axis']
+        for op_name in op_names:
+            # if input is bias which comes after conv, deconv, batchnorm operators
+            # then only reshape bias term
+            if inputs[0].name.startswith(op_name):
+                op_value = _fix_broadcast(current_op_name, inputs, broadcast_axis, proto_obj)
+                return op_value, new_attr, inputs
+    return current_op_name, new_attr, inputs
```

Comment:
So we are losing info about attrs here? We always return empty from this function. Sorry couldn't understand this logic.

Reply:
onnx attrs can have "broadcast/consumed_inputs" as input according to old onnx opset.. If broadcast is mentioned, we check that in line 253. But MXNet doesnt have these inputs.. broadcast ops in mxnet have empty attr. So by default passing empty.